### PR TITLE
Adding urfave/cli

### PR DESCRIPTION
**Pull request by creekorful**
Closes: #19

### CODE REVIEW

File: cmd/gopkg/gopkg.go [hunk 1]
```diff
@@ -1,48 +1,43 @@
...
-		err = build.Build(realPath)
-	default:
-		err = fmt.Errorf("unknow action")
+	app := cli.App{
+		Name:  "gopkg",
+		Usage: "Package manager for Golang written applications",
+		Authors: []*cli.Author{
+			{"Aloïs Micard", "alois@micard.lu"},
+			{"Fredrik Forsmo", "hello@frozzare.com"},
+			{"Johannes", "johannes@jitesoft.com"},
```
Comment (Johannestegner) [inline cmd/gopkg/gopkg.go:21]: Johannes Tegnér

;)

File: internal/cmd/build.go [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+	"fmt"
+	"github.com/go-pkg-org/gopkg/internal/build"
+	"github.com/urfave/cli/v2"
+	"os"
+	"path/filepath"
+)
+
+// ExecBuild execute the `gopkg build` command
+func ExecBuild(c *cli.Context) error {
+	if !c.Args().Present() {
```
Comment (Johannestegner) [inline internal/cmd/build.go:13]: Should we maybe just allow no arg and in that case use `os.GetWd()` or similar?

File: internal/cmd/make.go [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+import (
+	"fmt"
+	make2 "github.com/go-pkg-org/gopkg/internal/make"
+	"github.com/urfave/cli/v2"
+)
+
+// ExecMake execute the `gopkg make` command
+func ExecMake(c *cli.Context) error {
+	if !c.Args().Present() {
```
Comment (Johannestegner) [inline internal/cmd/make.go:11]: Same as above. Or maybe we just force them to use `.` in case they actually want to use the current dir.

**Review by Johannestegner — APPROVED** [review]
Looks good to me!

Left a few comments, but they are not something we _need_ to change, just thoughts :)

File: internal/cmd/build.go (hunk 2 above)
Comment (frozzare) [reply internal/cmd/build.go:13]: I think that's a good idea to fallback on working directory or something.

**Comment by frozzare** [comment]
Looks good to me, same thoughts as @Johannestegner has on allowing no args.

File: cmd/gopkg/gopkg.go (hunk 1 above)
Comment (creekorful) [reply cmd/gopkg/gopkg.go:21]: Sorry :o

File: internal/cmd/make.go (hunk 3 above)
Comment (creekorful) [reply internal/cmd/make.go:11]: Make only take importPath (for now)

File: internal/cmd/build.go (hunk 2 above)
Comment (creekorful) [reply internal/cmd/build.go:13]: I agree!

**Review by Johannestegner — APPROVED** [review]
LGTM!

**Pull request merged**